frontend/src/libs/components/dropdown-menu/styles.module.scss

```diff
@@ -74,11 +74,6 @@
     brightness(93%) contrast(86%);
 }
 
-.selected {
-  background-color: var(--gray-100);
-  cursor: pointer;
-}
-
```

Comment:
Should the selected option highlighting be removed totally? I think user have to see that and the problem was with different `pathname` due to the url params

Reply:
Good point

Reply:
Yep, thanks for the noticing. I will do it in this way